### PR TITLE
Jena 1723 or props

### PR DESCRIPTION
**Pull request by xristy**
PR for JENA-1723

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Jena README
+Jena README
```
Comment (ajs6f) [inline README.md]: Extra space?

File: jena-text/src/test/resources/log4j.properties [hunk 2]
```diff
@@ -12,5 +12,7 @@ log4j.logger.org.apache.jena.arq.exec=INFO
 # Reduce test/build noise
 log4j.logger.org.apache.jena.query.text.TextIndexLucene=ERROR
 
+# log4j.logger.org.apache.jena.query.text=TRACE
```
Comment (ajs6f) [inline jena-text/src/test/resources/log4j.properties]: Did you mean to leave this level of logging on for this PR?

Comment (xristy) [reply jena-text/src/test/resources/log4j.properties]: No I committed an update to log4j.properties @ 19:28Z to turn off the TRACE level.

Comment (ajs6f) [reply jena-text/src/test/resources/log4j.properties]: Cool!

File: README.md (hunk 1 above)
Comment (xristy) [reply README.md]: It is unintentional. A colleague was testing some github logic and apparently selected the jena fork's README.md. I've committed a repaired README.md

Comment (ajs6f) [reply README.md]: And it's gone.

**Comment by xristy** [comment]
Is one vote enough for me to go ahead and merge this PR?

**Comment by afs** [comment]
IMO yes - it is "lazy consensus". 

If you want specific review, you can request a review under "Reviewers".

@ajs6f comments are addressed as well, making 1.5 approvals :-)

**Comment by xristy** [comment]
Thanks @afs I just wanted to verify. I'll go ahead and merge.

**Pull request merged**